tools/eth-migration/README.md

```diff
@@ -0,0 +1,20 @@
+# Eth migrations tools
+Some scripts and tools to facilitate the key migration
+
+## Get Onchain msa id and keys
+`npm run control-keys --silent`
```

Comment:
nit: there is 1 line difference (the first one, a header) in output with/without `--silent`, and also, this command outputs the values but doesn't direct them to a file.

Reply:
I always preferred to have the option to pipe it to file in case or see it in console so that's why I didn't add the pipe to file.  But open to change it in case you have strong opinions about it